src/_includes/banner.html

```diff
@@ -5,6 +5,6 @@
     https://github.com/flutter/website/issues/3897
   {% endcomment %}
   <div class="site-banner site-banner--default" role="alert">
-    The next Flutter challenge is here! Join the <a href="https://flutterhack.devpost.com/">Flutter Puzzle Hack</a> for a chance to win prizes.
+    Exciting news for Windows developers&emdash;Flutter is on the way! <a href="https://flutter.dev/events/flutter-update-windows">Learn more</a>
```

Comment:
```suggestion
    Exciting news for Windows developers—Flutter is on the way! <a href="https://flutter.dev/events/flutter-update-windows">Learn more</a>
```

Reply:
I finally staged the PR and see this:

![image](https://user-images.githubusercontent.com/2164483/151445369-f6d36bca-fd0b-4fa7-a052-096d36f9f4a2.png)